pkg/swagger/swagger.go

```diff
@@ -74,9 +74,25 @@ func Run(api, outputDir string) error {
 		},
 	}
 
-	populateExamples(s.Paths)
+	if g.kubeConfig {
+		s.Paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RedHatOpenShift/openShiftClusters/{resourceName}/listAdminCredentials"] = &PathItem{
```

Comment:
Maybe move it closer to the list credentials as they are very similar? At some point we might want to move it inside a generator and I imagine they will go there together.

Reply:
done